html/canvas/tools/yaml-new/drawing-images-to-the-canvas.yaml

```diff
@@ -638,3 +638,11 @@
     @nonfinite ctx.drawImage(<bitmap>, <0 Infinity -Infinity NaN>, <0 Infinity -Infinity NaN>, <100 Infinity -Infinity NaN>, <50 Infinity -Infinity NaN>, <0 Infinity -Infinity NaN>, <0 Infinity -Infinity NaN>, <100 Infinity -Infinity NaN>, <50 Infinity -Infinity NaN>);
     @assert pixel 50,25 == 0,255,0,255;
   expected: green
+
+- name: 2d.drawImage.detachedcanvas
+  desc: drawImage with detached canvas as the source should throw exception
```

Comment:
Nit: this is really a detached OffscreenCanvas. Not sure whether more verbosity in the test name/description is warranted.